package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "lint": "semistandard",
-    "prepublish": "node downloadCurrentVersion.js && node verifyVersion.js",
+    "prepare": "node downloadCurrentVersion.js && node verifyVersion.js",
+    "prepublishOnly": "npm run prepare",
```

Comment:
New node separated the roles and warns when using the old role. However we do need to run it on both cases. I think this PR can wait. Need to figure out the node version this depends on.